app/views/nsm/steps/case_details/edit.html.erb

```diff
@@ -8,6 +8,14 @@
     <%= step_form @form_object do |f| %>
       <%= suggestion_select f, :main_offence, MainOffence.all, :name, :name, 'offence-autocomplete', width: 20,
                               options: { include_blank: true }, data: { autoselect: false }, label: { size: 's' } %>
+
+      <%= f.govuk_radio_buttons_fieldset :main_offence_type, legend: { size: 's', class: 'govuk-!-margin-bottom-6' } do %>
+        <% MainOffenceType.values.each_with_index do |main_offence_type, index| %>
```

Comment:
This isn't to spec (re: what html components it adds, see this [comment](https://github.com/ministryofjustice/laa-submit-crime-forms/pull/1310#discussion_r1852241256))

Reply:
That's only for when it's the only question on the page AFAICT

Reply:
true sorry